### ISSUE TITLE
focus accessible buffer doesn't work

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#178434]
- Create terminal
- Run navigate accessible buffer command
- Escape
- New split
- Shift+tab, second a11y buffer focused
- Shift+tab, first terminal focused
- Shift+tab, first a11y buffer NOT focused but should be
